docs/src/components/ComponentDoc/ComponentSourceManager/ComponentSourceManager.ts

```diff
@@ -0,0 +1,138 @@
+import * as _ from 'lodash'
+import * as React from 'react'
+
+import { ExampleSource } from 'docs/src/types'
+import formatCode from 'docs/src/utils/formatCode'
+import { componentAPIs as APIdefinitions, ComponentAPIs } from './componentAPIs'
+import getExampleSource from './getExampeSource'
+
+export type ComponentSourceManagerRenderProps = ComponentSourceManagerState & {
+  handleCodeAPIChange: (newApi: keyof ComponentAPIs) => void
```

Comment:
The Shorthand API should be the first selected option. This is the one we want to advertise. With the latest changes the Children API is first.

Reply:
Also, would recommend the Shorthand API to be the first option, as it was before.

Reply:
Agree, done with the latest commit :)